src/guides/v2.3/rest/operation-status-endpoints.md

```diff
@@ -51,6 +52,17 @@ Field name | Data type | Description
 }
 ```
 
+## Get operations count by bulk uuid and status.
+
+The `GET <host>/rest/<store_code>/V1/bulk/:bulkUuid/operation-status/:status` endpoint returns amount of operations from the bulk batch in particular status.
+
```

Comment:
Add a table that lists the possible status values. You can use the same text as above, except make it a simple table with 5 lines, rather than forcing line breakes with `<br/> elements.

Reply:
done